gazelle/pythonconfig/pythonconfig.go

```diff
@@ -78,6 +78,7 @@ const (
 	// GenerationModeProject defines the mode in which a coarse-grained target will
 	// be generated englobing sub-directories containing Python files.
 	GenerationModeProject GenerationModeType = "project"
+	GenerationModeFile    GenerationModeType = "file"
```

Comment:
Please document in `rules_python/gazelle/README.md` so that it would be visible in the table when viewing https://github.com/bazelbuild/rules_python/tree/main/gazelle